include/osg/StateSet

```diff
@@ -519,6 +519,8 @@ class OSG_EXPORT StateSet : public Object
         /** call release on all StateAttributes contained within this StateSet.*/
         virtual void releaseGLObjects(State* state=0) const;
 
+        /** generate Sampler StateAttributes from Textures. */
```

Comment:
This niche convenience function isn't appropriate for a general purpose class like osg::StateSet, it's presence bloats the API and could confuse users. Most OSG applications won't need to use osg::Sampler, it's not a feature that needs wide exposure.

If you want to have such a helper function then have a help function in the include/osg/Sampler header, or put into osgUtil.